src/components/offerPage/index.jsx

```diff
@@ -0,0 +1,356 @@
+                },
+              ]}
+            >
+              <PlacesAutocomplete
+                value={address}
+                onSelect={handleSelect}
+              >
+                {({
+                  getInputProps,
+                  suggestions,
+                  getSuggestionItemProps,
+                  loading,
+                }) => (
+                  <div>
+                    <Input
+                      {...getInputProps({ placeholder: "Enter your address" })}
+                    />
+
+                    <div className="description-list">
+                      {loading ? <div>...loading</div> : null}
+
+                      {suggestions.map((suggestion) => {
+                        return (
+                          <div
+                            {...getSuggestionItemProps(suggestion)}
+                            className="description-item"
+                          >
+                            {suggestion.description}
+                          </div>
+                        );
+                      })}
+                    </div>
+                  </div>
+                )}
+              </PlacesAutocomplete>
+            </Form.Item>
+            <Form.Item
+              className="input-to-hide"
+              name="settlement"
+              rules={[
+                {
+                  type: "string",
+                  message: inputValidationErrors.EMPTY_SETTLEMENT_MESSAGE,
+                },
+                {
+                  required: true,
+                  message: inputValidationErrors.EMPTY_SETTLEMENT_MESSAGE,
+                },
+              ]}
+            >
+              <Input name="settlement" type="hidden"  placeholder="Enter your settlement" />
+            </Form.Item>
+            <Form.Item
+              className="input-to-hide"
+              name="region"
+              rules={[
+                {
+                  type: "string",
+                  message: inputValidationErrors.EMPTY_SETTLEMENT_MESSAGE,
+                },
+                {
+                  required: true,
+                  message: inputValidationErrors.EMPTY_SETTLEMENT_MESSAGE,
+                },
+              ]}
+            >
+              <Input name="region" type="hidden" placeholder="Enter your region" />
+            </Form.Item>
+            <Form.Item
+              name="goodCategory"
+              rules={[
+                {
+                  required: true,
+                  message: inputValidationErrors.EMPTY_GOOD_CATEGORY_MESSAGE,
+                },
+              ]}
+            >
+              <Select placeholder="Select good categories">
+                {data?.map((res, idx) => (
+                  <Option value={res.name} key={idx}>
+                    {res.name.toLowerCase()}
+                  </Option>
+                ))}
+              </Select>
+            </Form.Item>
+
+            <Form.Item
```

Comment:
![Screenshot 2022-05-26 104334](https://user-images.githubusercontent.com/56841893/170442387-768b571c-9cad-483f-9b06-e0057cc33b49.jpg)

I can specify the time in the past